packages/components/molecules/f-mega-modal/README.md

```diff
@@ -54,54 +53,12 @@ export default {
         MegaModal: () => import(/* webpackChunkName: "mega-modal" */ '@justeat/f-mega-modal')
     }
 }
-
-```
-
-### Non-Vue Applications
-
-This module can be ran as a micro front-end for applications that don't make use of the Vue framework.
-
-The following rudimentary example can be used as a guide for implementing this component in an existing static application:
-
-```html
-<!doctype html>
-<html lang="en">
-<head>
-    <title>Mega Modal Example</title>
-    <link rel="stylesheet" href="https://unpkg.com/@justeat/f-mega-modal/dist/f-mega-modal.css">
-</head>
-<body>
-    <div data-app>
-        <mega-modal is-open>
-          <p>Modal content</p>
-        </mega-modal>
-    </div>
-    <script src="https://unpkg.com/vue@2.6.11/dist/vue.js"></script>
-    <script src="https://unpkg.com/@justeat/f-mega-modal/dist/f-mega-modal.umd.min.js"></script>
-    <script>
-        (function() {
-            if (typeof Vue === 'undefined') return null;
-
-            Vue.config.devtools = false;
-            Vue.config.productionTip = false;
-
-            return new Vue({
-                el: '[data-app]'
-            });
-    	})();
-    </script>
-</body>
-</html>
 ```
 
 ## Configuration
 
 ### Props
 
-`f-mega-modal` has a number of props that allow you to customise its functionality.
```

Comment:
I think this reads better as a README with this description (know it's a bit more verbose, but think it makes it more "friendly" as a README.

Reply:
I think it's totally redundant, properties are only there because they change functionality.

I understand that it's friendlier, but it's just more words to copy and copy and copy any nobody would think that it was rude without it.

Reply:
I don't have a strong opinion either way especially – but I would say that there's value in making a README more `human` and providing descriptions/intros to sections (it also breaks up the doc when reading through it – makes it less like a dry spec and more like a manual).

In reality, we don't copy this sort of structural text as it's auto-generated through the generator.

@justeat/ui-admins Any thoughts on this either way?

Reply:
If it's auto-generated I'd say leave it, otherwise, I'm don't mind either way really. I do agree that it's better to have readmes which are more friendly in tone as it makes the project more approachable for first-time users and contributors.

Reply:
Ah.  I was looking at these as specs, not a manual.  If it should be a manual than being more human is correct.